pages/charcuterie.js

```diff
@@ -76,6 +77,13 @@ export default function Charcuterie() {
         </ContentContainer>
         <Footer />
       </main>
+      <Carousel images={['https://upload.wikimedia.org/wikipedia/commons/thumb/1/11/Test-Logo.svg/783px-Test-Logo.svg.png', 'https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/dog-puppy-on-garden-royalty-free-image-1586966191.jpg', 'https://www.rd.com/wp-content/uploads/2021/01/GettyImages-588935825.jpg']}
+                height={180}
+                width={260} />
+
```

Comment:
Is there a reason for this duplicate `Carousel` component?

Reply:
no reason at all! Removed:)